README.md

```diff
@@ -99,7 +99,7 @@ Next, users will need to have the plugin within their Ratify pod in order to use
 One possible method to distribute plugins is by building a custom Ratify image
 
 ```Dockerfile
-FROM ghcr.io/deislabs/ratify:v1.0.0-beta.2 AS ratify
+FROM ghcr.io/ratify-project/ratify:v1.0.0-beta.2 AS ratify
```

Comment:
Update this to Ratify 1.3 once its available.

Reply:
Would do this in a separate PR